src/task/ReceiveTURNCredentialsTask.ts

```diff
@@ -89,9 +89,13 @@ export default class ReceiveTURNCredentialsTask extends BaseTask {
     this.context.turnCredentials = new MeetingSessionTURNCredentials();
     this.context.turnCredentials.password = responseBodyJson.password;
     this.context.turnCredentials.ttl = responseBodyJson.ttl;
-    this.context.turnCredentials.uris = responseBodyJson.uris.map((uri: string): string => {
-      return this.context.meetingSessionConfiguration.urls.urlRewriter(uri);
-    });
+    this.context.turnCredentials.uris = responseBodyJson.uris
+      .map((uri: string): string => {
+        return this.context.meetingSessionConfiguration.urls.urlRewriter(uri);
+      })
+      .filter((uri: string) => {
+        return uri !== '';
```

Comment:
- How about filtering out a falsy element? Note that `urlRewriter` returns `null` too. i.e. It will include `null` or `undefined`.

  ```
    urlRewriter: (url: string | null) => string | null = (url: string | null) => {
      return url;
    };
  ```

Reply:
Synced with you offline

Reply:
Anuran and I confirm that we will not handle other falsy cases in this PR. In case of an `null` or `undefined` element, SDK will anway fail to connect the session with a proper error message.